### DISCUSSION TITLE
Will osu! lazer support Windows 7 ever?

### DISCUSSION THREAD

**Discussion by deok420** [ppy/osu#20271] [Q&A] [answered]
I recently switched to Windows 7. I hate Windows 10, as it eats up a lot of ram. Is there any possibility that Windows 7 will be supported in the future? If yes, when?

**Comment by nagi-desuuu** [comment]
Osu targets .net 6.0, and for technical reasons, is not designed to be supported on windows 7 for a number of reasons, the most obvious one being, it's EOL!
At least, that's what I think.

**Comment by Theighlin** [comment, marked as answer]
Lazer can't and will most likely never be able to run on windows 7 as Realm doesn't support it.

**Reply by its-wawa** [reply]
Is there any supported version of the game I can play on Windows 7 SP1?
(with link of download please)

**Reply by peppy** [reply]
https://osu.ppy.sh/home/download
